cylc/flow/platforms.py

```diff
@@ -658,3 +662,30 @@ def get_localhost_install_target() -> str:
     """Returns the install target of localhost platform"""
     localhost = get_platform()
     return get_install_target_from_platform(localhost)
+
+
+def _validate(platforms) -> None:
+    """Check for invalid or inconsistent platforms config.
+
+    Some job runners require a single host (where job ID is only valid on the
+    specific submission host.)
+    """
+    bad_platforms = []
+    for name, deets in platforms.items():
```

Comment:
`deets` non obvious IMO - have used `spec` elsewhere, `config` would also be cool.

Reply:
:+1: haha, I didn't intend to leave that there - it was an on-the-fly sub for "details"